# New robot X position (m)
float32 x
# New robot Y position (m)
float32 y
# New robot Orientation (rads)
float32 orientation
---
bool ret 
